### PR TITLE
IDE-4075 add options "Dev Studio only" and "Dev Studio with cli"

### PR DESCRIPTION
**Pull request by jtydhr88**

### CODE REVIEW

File: build/installers/liferay-project-sdk-with-devstudio-ce/liferay-project-sdk-with-devstudio-ce.xml [hunk 1]
```diff
@@ -68,10 +68,230 @@
...
+                    </ruleList>
+                </runProgram>
+                <runProgram>
+                    <program>${bladeCommandPath}</program>
+                    <programArguments>--base "${lrws}" init</programArguments>
+                </runProgram>
+                <propertiesFileSet>
+                    <file>${lrws}${platform_path_separator}gradle.properties</file>
+                    <key>liferay.workspace.bundle.url</key>
+                    <value>https://cdn.lfrs.sl/releases.liferay.com/portal/7.1.0-a1/liferay-ce-portal-tomcat-7.1-a1-20180410105031513.zip</value>
```
Comment (gamerson) [inline build/installers/liferay-project-sdk-with-devstudio-ce/liferay-project-sdk-with-devstudio-ce.xml]: Update to the latest beta2

File: build/installers/liferay-project-sdk-with-devstudio-ce/liferay-project-sdk-with-devstudio-ce.xml [hunk 2]
```diff
@@ -68,10 +68,230 @@
...
+                <createDirectory>
+                    <path>${liferayBundlesPath}</path>
+                    <progressText>Creating directory...</progressText>
+                </createDirectory>
+                <copyFile>
+                    <destination>${liferayBundlesPath}${platform_path_separator}liferay-ce-portal-tomcat-7.1-a1-20180410105031513.zip</destination>
+                    <origin>${installdir}${platform_path_separator}liferay-ce-portal-tomcat-7.1-a1-20180410105031513.zip</origin>
+                </copyFile>
+                <copyFile>
+                    <destination>${liferayBundlesPath}${platform_path_separator}com.liferay.portal.plugins.sdk-1.0.16-withdependencies.zip</destination>
```
Comment (gamerson) [inline build/installers/liferay-project-sdk-with-devstudio-ce/liferay-project-sdk-with-devstudio-ce.xml]: Need to update the plugins.sdk with dependencies to latest version.

File: build/installers/liferay-project-sdk-with-devstudio-dxp/liferay-project-sdk-with-devstudio-dxp.xml [hunk 3]
```diff
@@ -68,36 +68,280 @@
...
+                    <actionList>
+                        <writeFile>
+                            <encoding>utf-8</encoding>
+                            <path>${installdir}${platform_path_separator}pwd</path>
+                            <text>${password.password}</text>
+                        </writeFile>
+                        <runProgram>
+                            <customErrorMessage>Could not create DXP download token. Make sure you type correct email address and password and are connected to the internet.</customErrorMessage>
+                            <program>${java_executable}</program>
+                            <programArguments>-jar "${installdir}${platform_path_separator}com.liferay.portal.tools.bundle.support.jar" createToken --force --email "${email}" --password-file "${installdir}${platform_path_separator}pwd"</programArguments>
```
Comment (gamerson) [inline build/installers/liferay-project-sdk-with-devstudio-dxp/liferay-project-sdk-with-devstudio-dxp.xml]: if there is proxyHost and proxy details you must pass them into this command.  `java -DproxyHost=.... -jar...`

File: build/installers/liferay-project-sdk-with-devstudio-dxp/liferay-project-sdk-with-devstudio-dxp.xml [hunk 4]
```diff
@@ -478,6 +489,30 @@
                 </if>
             </validationActionList>
         </directoryParameter>
+        <choiceParameterGroup>
+            <name>installcli</name>
+            <title>Install with cli tools?</title>
```
Comment (gamerson) [inline build/installers/liferay-project-sdk-with-devstudio-dxp/liferay-project-sdk-with-devstudio-dxp.xml]: Title should be `Installation Options`

File: build/installers/liferay-project-sdk-with-devstudio-ce/liferay-project-sdk-with-devstudio-ce.xml (hunk 2 above)
Comment (jtydhr88) [reply build/installers/liferay-project-sdk-with-devstudio-ce/liferay-project-sdk-with-devstudio-ce.xml]: hey Greg
the 1.0.16 is the latest one in https://repository.liferay.com/nexus/content/repositories/liferay-public-releases/com/liferay/portal/com.liferay.portal.plugins.sdk/

**Pull request closed without merging**